doc/manual/src/command-ref/env-common.md

```diff
@@ -103,15 +106,19 @@ Most Nix commands interpret the following environment variables:
     384 MiB. Setting it to a low value reduces memory consumption, but
     will increase runtime due to the overhead of garbage collection.
 
-## XDG Base Directory
+## XDG Base Directories
+
+Nix implements the [XDG Base Directory Specification].
```

Comment:
```suggestion
Nix follows the [XDG Base Directory Specification].
```
I think this is more correct; if anything it would be like the "desktop environment" that implements it, and applications are just supposed to go along with it.